vars/managerPipeline.groovy

```diff
@@ -97,6 +97,14 @@ def call(Map pipelineParams) {
                    description: 'If empty - the default manager version will be taken',
                    name: 'scylla_mgmt_repo')
 
+            string(defaultValue: "${pipelineParams.get('manager_branch', '')}",
+                   description: 'master_latest|2.4|2.3|2.2',
+                   name: 'manager_branch')
+
+            string(defaultValue: "${pipelineParams.get('target_manager_branch', '')}",
```

Comment:
this is for upgrade tests only, not all manager tests, right? shouldn't it be on the specific `jenkinsfile` instead the `groovy` file?
also, we can upgrade only to a newer version than set in the `manager_branch` parameter... so extra checks would be relevant in here, AFAIU

Reply:
It can't be just in the jenkinsfile, groovy is the code that reads the jenkinsfile, so it must be aware of this parameter

> also, we can upgrade only to a newer version than set in the manager_branch parameter... so extra checks would be relevant in here, AFAIU

True, but what extra checks can I do while in the groovy file? I only have access to the `defaults/manager_versions.yaml` while inside the code, not here